specifications/cdsc-wg1-01.md

```diff
@@ -1 +1,292 @@
+
+For invalid requests and error responses, Servers MUST respond with the appropriate Response Code for the type of issue or error (e.g. "503 Service Unavailable" if the Server is temporarily down for maintenance).
+
+### 3.1. Metadata Well-Known URI <a id="metadata-well-known-uri" href="#metadata-well-known-uri" class="permalink">🔗</a>
+
+If the Server wishes to provide metadata publicly, the Server MUST support a metadata endpoint at the Well-Known URI `"/.well-known/carbon-data-spec.json"` on their hostname (e.g. https://example.com/.well-known/carbon-data-spec.json).
+
+Servers MAY additionally support other public and restricted URLs for special case or private metadata endpoints, as needed.
+
+### 3.2. Metadata Object Format <a id="metadata-object-format" href="#metadata-object-format" class="permalink">🔗</a>
+
+Metadata objects are formatted as JSON objects and contain named values, some of which are required and some of which are optional. The following values are included in the default list available in metadata objects.
+
+* `cds_metadata_version` - _string_ - (REQUIRED) The version of metadata object format this Server follows, which for this version of the specification is `v1`
+* `cds_metadata_url` - _string_ - (REQUIRED) A URL link to this metadata object's canonical network location
+* `created` - _ISO8601 datetime_ - (REQUIRED) When the metadata url was first made available
+* `updated` - _ISO8601 datetime_ - (REQUIRED) When this metadata object was last modified
+* `name` - _string_ - (REQUIRED) The name of the utility or entity (e.g. "Demo Gas & Electric")
+* `description` - _string_ - (REQUIRED) A brief description about the utility or entity (e.g. "An electric and gas utility serving customers in Northern California")
+* `website` - _URL_ - (REQUIRED) Where users can learn more about the utility or entity and its capabilities
+* `documentation` - _URL_ - (REQUIRED) Where Clients can find the utility or entity's technical reference documentation
+* `support` - _URL_ - (REQUIRED) Where Clients can find contact information for technical support on the Server's capabilities
+* `infrastructure_types` - _Array[[InfrastructureType](#infrastructure-types)]_ - (REQUIRED) A list of infrastructure categories that are applicable to this utility or entity, which may be an empty array if the Server is not any of the possible infrastructure types
+* `commodity_types` - _Array[[CommodityType](#commodity-types)]_ - (REQUIRED) A list of services or commodities that are applicable to this utility or entity, which may be an empty array if the Server does not have an applicable commodity
+* `capabilities` - _Array[[ServerCapability](#server-capabilities)]_ - (REQUIRED) What capabilities and functionality this Server supports, which may be an empty array if the Server is not offering any additional capabilities beyond just this metadata object
+* `coverage` - _URL_ - (OPTIONAL) Where to find the [Coverage Endpoint](#coverage-endpoint)
```

Comment:
From @halliecramer1 ([on April 14, 2023](https://github.com/carbon-data-specification/Customer-Data/pull/23#discussion_r1167326314)):

---

Suggest restructuring coverage - instead of as a capability, create a `coverage_types` array, and for each coverage type in that list, you have a (nested?) entry format. For example, `territory` is one coverage type, and then you nest the Coverage Entry Format - which could have fields specific to that type - but `sub_type` would replace `type` and refer to the service type (e.g. `electric_service`) instead of to territory as a type. Other coverage types could include: `customer_class`, and/or `rate_class` where `sub_type` is e.g. residential / commercial / industrial / client (in case of `energy_solutions_provider`), and/or the specific tariff name. 

Then for each capability, you can include a list of coverage subtypes that it applies to (or list of type, subtype key-value pairs).

---